src/common/options/rgw.yaml.in

```diff
@@ -3618,6 +3618,7 @@ options:
   - none
   - base
   - d4n
+  - posix
```

Comment:
part of rgw_config_store?

Reply:
I don't understand the question?  In the future (work Ali is working on) all this will be replaced with JSON stored in the zonegroup, but for now it has to live in ceph.conf.

Reply:
i don't think it's necessary to have a posix-file backend for `rgw::sal::ConfigStore`. `rgw_config_store=dbstore` already works on a local filesystem

Reply:
is "JSON stored in the zonegroup" equivalent to putting things in sqlite?  I don't follow

Reply:
> is "JSON stored in the zonegroup" equivalent to creating a posix-file backend for rgw::sal::ConfigStore? I don't follow

No, it describes the driver stack, so it's a replacement for `rgw_backend_store` and `rgw_filter` currently in ceph.conf.  The actual zonegroup is stored in the ConfigStore, so whichever ConfigStore you use will store the JSON that describes the driver stack.  I agree with Casey, we'll probably use DBStore's ConfigStore for POSIXDriver.

Reply:
> part of rgw_config_store?

Ah, was the original question asking if this should be in rgw_config_store instead of rgw_filter?  If so, then yes, eventually.  Currently, POSIXDriver is a filter, since it doesn't implement any User APIs, but instead depends on DBUser.  So it has to be a filter stacked on top of DBStore until we finish the User APIs.